Scripts/CI/CreateBuildDrop.bat

```diff
@@ -0,0 +1,33 @@
+@ECHO OFF
+CALL %~dp0\..\InitializeEnvironment.bat || EXIT /b 10
+
+IF "%1"=="" GOTO USAGE
+IF "%2"=="" GOTO USAGE
+
+SETLOCAL enableextensions
+SET Configuration=%1
+SET VFS_STAGEDIR=%2
+
+REM Prepare the staging directories for functional tests.
+IF EXIST %VFS_STAGEDIR% (
+  rmdir /s /q %VFS_STAGEDIR%
+)
+mkdir %VFS_STAGEDIR%\src\Scripts
+mkdir %VFS_STAGEDIR%\BuildOutput\GVFS.Build
+mkdir %VFS_STAGEDIR%\BuildOutput\GVFS.FunctionalTests\bin\x64\%Configuration%\netcoreapp2.1
+mkdir %VFS_STAGEDIR%\BuildOutput\GVFS.FunctionalTests.Windows\bin\x64\%Configuration%\

```

Comment:
Wondering if we really need this deep of a folder structure?  Aren't we already naming the stage directory for the configuration and if bin and x64 are hard coded, do we need those?  Could we use `BuildOutput\GVFS.FunctionalTests.NetCore` and `BuildOutput\GVFS.FunctionalTests.Windows` and remove all the intermediary folders?

Reply:
I think the idea is that it should match the same layout as if you had run a build on that machine? That way the test scripts/tools don't have to change their understanding of where the binaries live.

Reply:
@sanoursa is exactly right, @kewillford. This means the scripts run identically whether you built this from source on your box or if you downloaded the drop.